src/Surfnet/StepupGateway/GatewayBundle/Resources/views/SecondFactor/verifySmsSecondFactor.html.twig

```diff
@@ -9,7 +9,6 @@
 
     <ol>
         <li>{{ 'gateway.second_factor.sms.text.ensure_phone_has_signal'|trans }}</li>
-        <li>{{ 'gateway.second_factor.sms.text.enter_phone_number_below'|trans }}</li>
```

Comment:
Manually removed the now unused "gateway.second_factor.sms.text.enter_phone_number_below" from the xliff